### ISSUE TITLE
SSL causing the crashing issues

### ISSUE THREAD

**Issue by shanep2300** [iandwelker/smserver#33]
I believe this will fix the 2 open issues #28 and #18 . When I turned off Enable SSL, everything works perfectly. Backgrounding, chats, mark as read, etc. Not sure how I can trouble shoot this for you as I don't understand how SSL works...

On the latest versions of the apps. MRYIPC is back down to 1.0, libsmserver 0.3.3, SMServer 0.4.0

**Comment by iandwelker** [comment]
So just to clarify: You run into issues with the server crashing when you click on a chat thread & not backgrounding for forever, but when you disable SSL, you don't run into either of those issues?

Also, somewhat unrelated: it's safe to upgrade to libmryipc 2.0 now; I fixed all the issues that made SMServer incompatible with it.

**Comment by shanep2300** [comment]
Originally I had Enable Background, Enable SSL, Mark conversations read, turned on. This was causing background issues and the server crashing every time I tried accessing the page. FireFox kept on giving me the notification to accept the cert on the 8740 port, but nothing would happen or pop up. Updated to 0.4.0 this morning, and only turned of SSL and now it's always in background and working fine.

As for MRYIPC, it downgraded on its own. There is no update to 2.0 for me. Think dev did that.

Not sure if this is SSL related, but I'm not getting notifications from the web interface and since I have Mark conver as read, my phone doesn't get the notification either.
